src/GeositeFramework/css/main.css

```diff
@@ -824,6 +824,27 @@ nav {
     width: 28px;
 }
 
+.nav-apps-narrow {
```

Comment:
I'd rather not have to duplicate all of these rules, but I couldn't think of a way to have inside and outside of the media query below.